src/components/Search/Search.scss

```diff
@@ -0,0 +1,20 @@
+.search {
+  @include mdc-text-field-label-color(rgba(128, 128, 128, 0.719));
```

Comment:
where does this color come from? maybe there is a variable in mdc for this?

Reply:
This is the custom color I set here ( https://screencast.com/t/TBVa9Wmaey ). I will specify it like #c0c0c0 - almost the same color. Just simpler. Will it be Ok?